schutzbot/mockbuild.sh

```diff
@@ -69,13 +69,13 @@ greenprint "📤 RPMS will be uploaded to: ${REPO_URL}"
 # Build source RPMs.
 greenprint "🔧 Building source RPMs."
 make srpm
-# rhel 8.4 will run off of the nightly repos and does not have a redhat subscription
-if [[ $VERSION_ID == 8.4 ]]; then
+# rhel 8.4 and 8.5 will run off of the internal repos and does not have a redhat subscription
+if [[ $VERSION_ID == 8.4 || $VERSION_ID == 8.5 ]]; then
     greenprint "📋 Updating RHEL 8 mock template for unsubscribed image"
     sudo sed -i '/# repos/q' /etc/mock/templates/rhel-8.tpl
     # remove the subscription check
     sudo sed -i "s/config_opts\['redhat_subscription_required'\] = True/config_opts['redhat_subscription_required'] = False/" /etc/mock/templates/rhel-8.tpl
-    cat "$RHEL84_NIGHTLY_REPO" | sudo tee -a /etc/mock/templates/rhel-8.tpl > /dev/null
```

Comment:
Hmm, shouldn't we use different repos for 8.4 and 8.5? Another option is to turn off 8.4 mockbuild because we don't expect any more changes there (it's now too late in the process).

Reply:
Ahhhhhh, I see what you did... Clever... I think I would prefer naming the variable literally because 1) it confused me :-D 2) no all CI systems support this (e.g. GitLab)

Reply:
heh, fair enough.  I did the same on composer so I'll change it there too.

Reply:
I duplicated the whole code block in the condition to make it easier to remove 8.4 completely.  Unless you want it removed now?